src/index.d.ts

```diff
@@ -0,0 +1,40 @@
+declare namespace API {
```

Comment:
This schema does not match the API definitions exactly. Something to keep in mind and unify later in the project when we start talking to the API.